pkg/kubelet/prober/worker.go

```diff
@@ -251,7 +251,7 @@ func (w *worker) doProbe() (keepGoing bool) {
 	if c.Started != nil && *c.Started {
 		// Stop probing for startup once container has started.
```

Comment:
please update comment to indicate that we keep it running to make sure it will work for restarted container